src/clue/components/teacher/teacher-group-six-pack.tsx

```diff
@@ -42,13 +42,14 @@ export class TeacherGroupSixPack extends BaseComponent<IProps, IState> {
   private renderGroups() {
     const { page, documentViewMode, selectedSectionId } = this.props;
     const { groups } = this.stores;
-    const numberOfGroups = groups.allGroups.length;
     const renders = [];
+    const renderGroups = groups.allGroups.filter(group => group.users.length > 0);
```

Comment:
Is this right?  It makes sense to me to filter groups with no members.  But could it be useful for a teacher to see a member-less group as part of some class setup step (determining which groups exist so a teacher can decide which group a new user might be added to or something like that)?

Reply:
We can check, but I think so. It came up because there's an empty group 40 that keeps showing up in demo class 1. Since students set up classes, not teachers, I don't think there's anything related to class setup that would play into it.